cli/deployment/config.go

```diff
@@ -499,10 +499,11 @@ func newConfig() *codersdk.DeploymentConfig {
 			Default: flag.Lookup("test.v") == nil && !buildinfo.IsDev(),
 		},
 		MaxTokenLifetime: &codersdk.DeploymentConfigField[time.Duration]{
-			Name:    "Max Token Lifetime",
-			Usage:   "The maximum lifetime duration users can specify when creating an API token.",
-			Flag:    "max-token-lifetime",
-			Default: 24 * 30 * time.Hour,
+			Name:  "Max Token Lifetime",
+			Usage: "The maximum lifetime duration users can specify when creating an API token.",
+			Flag:  "max-token-lifetime",
+			// max time.Duration is 290 years
+			Default: 290 * 365 * 24 * time.Hour,
```

Comment:
According to t[he docs](https://pkg.go.dev/time#Duration), the max duration one can set is 290 years. If there's a cleaner way to represent this max, let me know.

Reply:
A cleaner way to do this is `time.Duration(math.MaxInt64)`

Reply:
Thanks! I'll follow up in my next PR!

Reply:
for fun: https://go.dev/play/p/YtHsPbOIR-N